app/controllers/npcs_controller.rb

```diff
@@ -1,10 +1,10 @@
 class NpcsController < ApplicationController
   def target
     target = Npc.ensure(params[:id])
-    raise InvalidRequest.new( time: current_user.active_spaceship.get_target_time ) unless target
-    raise InvalidRequest.new( time: current_user.active_spaceship.get_target_time ) unless current_user.can_be_attacked?
-    raise InvalidRequest.new( time: current_user.active_spaceship.get_target_time ) unless target.location_id == current_user.location_id
-    raise InvalidRequest.new( time: current_user.active_spaceship.get_target_time ) unless current_user.npc_target_id != target.id
+    raise InvalidRequest.new(time: current_user.active_spaceship.get_target_time) unless target
+    raise InvalidRequest.new(time: current_user.active_spaceship.get_target_time) unless current_user.can_be_attacked?
+    raise InvalidRequest.new(time: current_user.active_spaceship.get_target_time) unless target.location_id == current_user.location_id
+    raise InvalidRequest.new(time: current_user.active_spaceship.get_target_time) unless current_user.npc_target_id != target.id
```

Comment:
Can't we unify this into a single raise?

```
unless target || current_user.can_be_attacked? || ...
  raise InvalidRequest
end
```

Reply:
unless with multiple conditions is a tricky thing... but will see

Reply:
You can also use

```
if !target || !current_user.can_be_attacked? || ...
  raise InvalidRequest
end
```

but I think that DRY is much more heavier here than the Ruby Style Guide :)

Reply:
also... your monitor must be a lot wider than mine, because that line goes waaaay off the right edge of my editor :)

Reply:
Can't you do

```
if !target ||
  !current_user.can_be_attacked? ||
  ...
  raise Invalid Request
end
```

? I don't know if that is working

Reply:
Also, are there other places in which there are multiple raises that can be combined?